### PR TITLE
Added NL (Dutch) translation

### PR DESCRIPTION
**Pull request by ronvandegraaf**
Dutch translation added. Strings are up to date with the current English translation.

### CODE REVIEW

File: static/build/miyoo/app/lang/nl.lang [hunk 1]
```diff
@@ -0,0 +1,135 @@
+{
+	"lang":	"Nederlands",
+	"0":	"Expert",
+	"1":	"Favorieten",
+	"2":	"GAMES",
```
Comment (Aemiii91) [inline static/build/miyoo/app/lang/nl.lang:5]: Should not be all caps

File: static/build/miyoo/app/lang/nl.lang [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+	"114":	"Uit",
+	"115":	"Slaaptimer",
+	"116":	"Game van SD-kaart verwijderen?",
+	"117":	"Bevestig dat je\ndeze game van je SD-kaart wilt verwijderen.",
+	"118":	"Weergave",
+	"119":	"Luminantie",
+	"120":	"Tint",
+	"121":	"Verzadiging",
+	"122":	"Contrast",
+	"123":	" \n \nWeergave-instellingen\nopgeslagen.\n \n ",
```
Comment (Aemiii91) [inline static/build/miyoo/app/lang/nl.lang:126]: Can you add a note, asking people to reboot the device (new for V4.1)

**Review by Aemiii91 — CHANGES_REQUESTED** [review]

File: static/build/miyoo/app/lang/nl.lang (hunk 2 above)
Comment (ronvandegraaf) [reply static/build/miyoo/app/lang/nl.lang]: Added reboot note.

File: static/build/miyoo/app/lang/nl.lang (hunk 1 above)
Comment (ronvandegraaf) [reply static/build/miyoo/app/lang/nl.lang]: Changed it.

**Review by Aemiii91 — APPROVED** [review]

**Comment by Aemiii91** [comment]
Thank you very much!

**Pull request merged**